README.md

```diff
@@ -1,5 +1,62 @@
 # OpenTelemetry Go exporter for Dynatrace
 
-This project is currently still under development and should not be used yet.
+This is the home of the OpenTelemetry Go exporter for Dynatrace. This package allows you to export spans to Dynatrace
+using the OpenTelemetry API.
 
-We will not provide any support or accept external contributions before the first official release of this package.
+This package is provided as a "core package", meaning that no platform-specific span attributes are automatically
+provided, which might be required by Dynatrace in order to enable service detection and correlation.
+
+Please refer to the [Dynatrace documentation](tbd) for how to set up tracing for Google Cloud Functions with Go using
+this package.
+
+## Prerequisites
+
+You need a Dynatrace environment to export your spans to.
+
+For platform-specific prerequisites, please refer to [Prerequisites](tbd).
+
+## Getting started
+
+Follow [the instructions](https://www.dynatrace.com/support/help/setup-and-configuration/setup-on-cloud-platforms/google-cloud-platform/opentelemetry-integration/opentelemetry-on-gcf#choose-config-method)
+on how to set up the Dynatrace configuration for your project by using a config file or environment variables. An error will be returned during the TracerProvider instantiation if the config cannot be found.
+
+1. Import the package into your Go project:
+    ```shell
+    go get github.com/dynatrace-oss/opentelemetry-exporter-go/core
+    ```
+2. Create a Dynatrace TracerProvider and TextMapPropagator:
+    ```go
+    import (
+        "go.opentelemetry.io/otel"
+        dtTrace "github.com/dynatrace-oss/opentelemetry-exporter-go/core/trace"
+    )
+   
+    // create a TracerProvider
+    tracerProvider, err := dtTrace.NewTracerProvider()
```

Comment:
Also show the call to `Shutdown` please, maybe even with `defer`. That way we prevent spans from getting lost.